File: ReleaseNotes.API/ReleaseNotes.API/Services/Bundle/Models/BundleReleaseTimeRangesResponseModel.cs

using ReleaseNotes.API.Services.Project.Models.ReleaseNotes;

namespace ReleaseNotes.API.Services.Bundle.Models;

public class BundleReleaseTimeRangesResponseModel
{
    public DateTimeOffset? CreatedOnUtc { get; set; }
    public string Version { get; set; } = string.Empty;
    public DateTimeOffset? StartTimeUtc { get; set; }
    public DateTimeOffset? EndTimeUtc { get; set; }
    public List<ReleaseNotesResponseModel> ReleaseNote { get; set; } = [];
    public Guid Id { get; internal set; }
    public string BundleName { get; internal set; } = string.Empty;
}

File: ReleaseNotes.UI/src/services/bundle/models/bundle-release-time-ranges-response-model.ts
import { ReleaseNotesResponseModel } from "../../project/models/release-notes/release-notes-response-model";

export interface BundleReleaseTimeRangesResponseModel {
  id: string;
  CreatedOnUtc: Date;
  version: string;
  startTimeUtc: string;
  endTimeUtc: string;
  releaseNote:  ReleaseNotesResponseModel[];
}
